js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadAllowedRoot.tsx

```diff
@@ -86,17 +86,17 @@ export const LaunchpadAllowedRoot = (props: Props) => {
       return undefined;
     }
 
+    if (!pipelineOrError || pipelineOrError.__typename !== 'Pipeline') {
+      return undefined;
+    }
+
     const rootDefaultYaml = runConfigSchemaOrError.rootDefaultYaml;
-    const opNameList = sessionPresets?.assetSelection
-      ? sessionPresets.assetSelection.map((entry) => entry.opNames ?? []).flat()
-      : [];
+    const opNameList = pipelineOrError?.nodeNames ?? [];
```

Comment:
nit: we should rename this to nodeNameList, because the nodeNames will include graph names as well